### PR TITLE
change prepend_op to append_op in initializer

### PR DESCRIPTION
**Pull request by zhiqiu**
### PR types

Breaking changes
### PR changes

Others
### Describe

change prepend_op to append_op in initializer to make dygraph and static graph consistent

Given the following code, with the same random SEED=123, the same parameter initialized in static graph and dynamic graph is different.

```
import paddle
paddle.set_device('cpu')
SEED = 123

weight_attr=paddle.framework.ParamAttr(name="linear_weight", learning_rate=1.0,
        trainable=False, regularizer=None, initializer=paddle.nn.initializer.TruncatedNormal(mean=0.0, std=2.0))
bias_attr=paddle.framework.ParamAttr(name="linear_bias", learning_rate=1.0,
        trainable=False, regularizer=None, initializer=paddle.nn.initializer.TruncatedNormal(mean=0.0, std=2.0))

if True:
    print('Test dynamic graph')
    paddle.seed(SEED)
    linear = paddle.nn.Linear(1, 1, weight_attr=weight_attr, bias_attr=bias_attr)

    print('linear.weight: ', linear.weight.numpy())
    print('linear.bias: ', linear.bias.numpy())

if True:
    print('Test static graph')
    paddle.enable_static()

    exe = paddle.static.Executor(paddle.CPUPlace())
    paddle.seed(SEED)

    linear = paddle.nn.Linear(1, 1, weight_attr=weight_attr, bias_attr=bias_attr)

    res = exe.run(paddle.static.default_startup_program(), fetch_list=['linear_weight', 'linear_bias'])

    print('linear.weight: ', res[0])
    print('linear.bias: ', res[1])
```

- Before
```
Test dynamic graph
linear.weight:  [[1.594699]]
linear.bias:  [3.8483276]
Test static graph
linear.weight:  [[3.8483276]]
linear.bias:  [1.594699]
```
- After
```
Test dynamic graph
linear.weight:  [[1.594699]]
linear.bias:  [3.8483276]
Test static graph
linear.weight:  [[1.594699]]
linear.bias:  [3.8483276]
```

### CODE REVIEW

**Review by Aurelius84 — APPROVED** [review]
LGTM

**Review by phlrain — APPROVED** [review]
